app/domain/invoices/abacus/course_annulation_invoice.rb

```diff
@@ -17,6 +17,8 @@ def additional_user_fields
       private
 
       def position_description_and_amount
+        return [I18n.t("cancellation_costs", scope: "invoices.course_annulation", percentage: 100), @invoice_total] if @invoice_total
```

Comment:
Da der ganze Aufbau (Array, Übersetzungen, Percentage) von CourseAnnulationCost kommt, würde ich den @custom_price auch gleich dort als Argument rein geben und die Logik dort platzieren. Dann muss in dieser Methode gar nichts angepasst werden.